src/Compat/SearchControls.php

```diff
@@ -259,4 +257,23 @@ protected function enrichFilterCondition(Filter\Condition $condition, Query $que
             $condition->metaData()->set('columnLabel', $label);
         }
     }
+
+    /**
+     * Decode the given param names from the given Url
+     *
+     * @return array<string, mixed> decoded key => value pairs
```

Comment:
Please also mark this as `@internal`, I don't want to be responsible to keep compatibility here.